Tools/scripts/checkpyc.py

```diff
@@ -51,7 +51,7 @@ def main():
                     print('Bad MAGIC word in ".pyc" file', end=' ')
                     print(repr(name_c))
                     continue
-                mtime = get_long(mtime_str)
+                mtime = get_long(mtime_str) & 0xFFFF_FFFF
                 if mtime in {0, -1}:
```

Comment:
mtime can't be -1 after adding `& 0xFFFF_FFFF`.

Reply:
maybe add the 0xFFFF_FFFF on the line 57?

Reply:
Yes.